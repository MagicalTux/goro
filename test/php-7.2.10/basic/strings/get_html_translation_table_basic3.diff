     *** Testing get_html_translation_table() : basic functionality ***
     -- with table = HTML_SPECIALCHARS & quote_style = ENT_COMPAT --
003- array(4) {
004-   ["&"]=>
005-   string(5) "&amp;"
006-   [">"]=>
007-   string(4) "&gt;"
008-   ["<"]=>
009-   string(4) "&lt;"
010-   ["""]=>
011-   string(6) "&quot;"
012- }
013- -- with table = HTML_SPECIALCHARS & quote_style = ENT_QUOTES --
014- array(5) {
015-   ["'"]=>
016-   string(6) "&#039;"
017-   ["&"]=>
018-   string(5) "&amp;"
019-   [">"]=>
020-   string(4) "&gt;"
021-   ["<"]=>
022-   string(4) "&lt;"
023-   ["""]=>
024-   string(6) "&quot;"
025- }
026- -- with table = HTML_SPECIALCHARS & quote_style = ENT_NOQUOTES --
027- array(3) {
028-   ["&"]=>
029-   string(5) "&amp;"
030-   [">"]=>
031-   string(4) "&gt;"
032-   ["<"]=>
033-   string(4) "&lt;"
034- }
035- Done
003+ 
004+ Fatal error: Uncaught Error: Call to undefined function get_html_translation_table in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/get_html_translation_table_basic3.php:15
005+ Stack trace:
006+ #0 {main}
007+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/get_html_translation_table_basic3.php on line 15
