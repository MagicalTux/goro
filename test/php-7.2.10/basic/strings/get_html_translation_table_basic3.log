
---- EXPECTED OUTPUT
*** Testing get_html_translation_table() : basic functionality ***
-- with table = HTML_SPECIALCHARS & quote_style = ENT_COMPAT --
array(4) {
  ["&"]=>
  string(5) "&amp;"
  [">"]=>
  string(4) "&gt;"
  ["<"]=>
  string(4) "&lt;"
  ["""]=>
  string(6) "&quot;"
}
-- with table = HTML_SPECIALCHARS & quote_style = ENT_QUOTES --
array(5) {
  ["'"]=>
  string(6) "&#039;"
  ["&"]=>
  string(5) "&amp;"
  [">"]=>
  string(4) "&gt;"
  ["<"]=>
  string(4) "&lt;"
  ["""]=>
  string(6) "&quot;"
}
-- with table = HTML_SPECIALCHARS & quote_style = ENT_NOQUOTES --
array(3) {
  ["&"]=>
  string(5) "&amp;"
  [">"]=>
  string(4) "&gt;"
  ["<"]=>
  string(4) "&lt;"
}
Done
---- ACTUAL OUTPUT
*** Testing get_html_translation_table() : basic functionality ***
-- with table = HTML_SPECIALCHARS & quote_style = ENT_COMPAT --

Fatal error: Uncaught Error: Call to undefined function get_html_translation_table in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/get_html_translation_table_basic3.php:15
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/get_html_translation_table_basic3.php on line 15
---- FAILED
