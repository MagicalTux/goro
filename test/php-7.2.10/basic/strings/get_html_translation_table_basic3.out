*** Testing get_html_translation_table() : basic functionality ***
-- with table = HTML_SPECIALCHARS & quote_style = ENT_COMPAT --

Fatal error: Uncaught Error: Call to undefined function get_html_translation_table in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/get_html_translation_table_basic3.php:15
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/get_html_translation_table_basic3.php on line 15